astrokat/correlator.py

```diff
@@ -1,10 +1,12 @@
 # Setting correlator configuration values different from default values
 
+
 def set_fengines(session,
-        requant_gains=None,
-        fft_shift=None):
+                 requant_gains=None,
+                 fft_shift=None):
+    msg = 'Failed to get correlator input labels: '
     if not session.cbf.fengine.inputs:
-        msg = 'Failed to get correlator input labels, cannot set the F-engine gains'
+        msg += 'cannot set the F-engine gains'
```

Comment:
This change seems unnecessary.  The base `msg` isn't used anywhere else - lines 17 and 24 reassign it.

Reply:
Cleaned up implementation to single message